### PR TITLE
style:Create the idea Card in profile

### PR DESCRIPTION
**Pull request by EsraaY21**
issue #16

### CODE REVIEW

**Comment by EsraaY21** [comment]
Note: To see the changes go to `/account`

**Review by aryankarim — DISMISSED** [review]
Good work! let us know if you have any problems resolving those conflicts!

**Comment by EsraaY21** [comment]
@aryanmajeed Thank you so much! I will =D

File: public/images/pexels-photo-355948 2.png:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (MuhannedNoman) [inline public/images/pexels-photo-355948 2.png:Zone.Identifier]: Didn't those .Identifier files cause issues as @HaiderAltufaily noted in #33 ?

File: components/Account/IdeaCard.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { Box, Link, Image, Text, HStack, Flex } from "@chakra-ui/react";
+import { Image as NextImage } from "next/image";
+
+const IdeaCard = ({ ideaImage, title, userImage, username, date }) => {
```
Comment (MuhannedNoman) [inline components/Account/IdeaCard.js]: Can we also add the href of the project so that when we click on the card we go there?

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
Nice work here @EsraaY21 Can you take a look at the comments I left here and le me know what you think!

File: public/images/pexels-photo-355948 2.png:Zone.Identifier (hunk 1 above)
Comment (IsraaY21) [reply public/images/pexels-photo-355948 2.png:Zone.Identifier]: Yes, I will remove them.

File: components/Account/IdeaCard.js (hunk 2 above)
Comment (IsraaY21) [reply components/Account/IdeaCard.js]: Ok, I will do that.

**Review by MuhannedNoman — DISMISSED** [review]
LGTM, but is the about page and card supposed to be in this PR?

**Comment by EsraaY21** [comment]
I already did the about page and merged it but then I wanted to do very small changes so I thought I could do that here. I forgot that I needed to create an issue for that.

**Review by aryankarim — APPROVED** [review]

**Pull request merged**